app/models/deed.rb

```diff
@@ -13,6 +13,21 @@ class Deed < ActiveRecord::Base
   TRANSLATION_INDEXED = 'xlat_index'
   WORK_ADDED = 'work_add'
 
+  CONTRIBUTOR_DEED_TYPES = [
+    PAGE_TRANSCRIPTION,
+    PAGE_EDIT,
+    PAGE_INDEXED,
+    ARTICLE_EDIT,
+    NOTE_ADDED,
+    PAGE_TRANSLATED,
+    PAGE_TRANSLATION_EDIT,
+    OCR_CORRECTED,
+    NEEDS_REVIEW,
+    TRANSLATION_REVIEW,
+    TRANSLATION_INDEXED
+  ]
+
+
```

Comment:
Didn't we already have CONTRIBUTOR_DEED_TYPES ?  Why did we have to add it here?

Reply:
Not that I was able to find.  Where have you seen it?

Reply:
I don't think we have contributor_deed_types, but deeds are already associated with user_ids.  What are these new contributor_deed_types used for?